collect_app/src/main/java/org/odk/collect/android/views/MediaLayout.java

```diff
@@ -142,9 +142,9 @@ public void playVideo() {
 
             Intent i = new Intent("android.intent.action.VIEW");
             i.setDataAndType(Uri.fromFile(videoFile), "video/*");
-            try {
+            if (i.resolveActivity(getContext().getPackageManager()) != null) {
```

Comment:
That seems possible, I haven't seen it before, strangely. I suppose it would only happen when someone reuses a branch (it's better not to!). 

@nikhilbalyan Last thing, you have some conflicts with master. So do try to merge master in and hopefully that will address this as well.